README.md

```diff
@@ -1,81 +1,95 @@
+R is an Open Source programming language for data analysis. When data science meets education, practitioners can use the numbers previously confined to websites and PDF reports. Teachers, administrators, and consultants can apply programming and statistics to prepare data, transform it, visualize it, and analyze it. These practices empower practitioners to answer questions that make a difference for their students.
 
-Our book focuses on *data science in education*, which we define as using data science techniques like preparing, exploring, visualizing, and modeling data, in order to support schooling at all levels. We want to make a case for learning about data science through field-specific examples. Understanding the unique challenges and starting to use a common field-specific language is important for mastering data science in education. We feel that discussing data science using education-specific scenarios more effectively speaks to the needs of educators.
+Our book focuses on data science in education, which we define as using data science techniques to support schooling at all levels. These techniques include preparing, exploring, visualizing, and modeling data. 
 
-Technology is transforming both the administrative and student-facing sides of education. It's becoming increasingly important for educators - not just people hired to analyze data - to understand what stories this new data tells them them about their students. Our book empowers educators from elementary school to higher education to transform educational data into actionable insights so it helps them serve their students and institutions. We wrote our book to be used as a main textbook in a graduate data science in education course. We also wrote it as a practical reference for data scientists working with education data.
+These techniques shouldn't be learned separately from education use cases. We propose learning about data science through field-specific examples. Using common field-specific language is important for learning techniques that are practical for the job. We feel that discussing data science using education-specific scenarios will make learning more fun and meaningful.
+
+Technology is transforming education for administrators, staff, and students. It is increasingly important for educators -- not just data analysts -- to use data to reveal the stories of their students. Our book empowers educators from elementary school to higher education to transform educational data into actionable insights. We wrote our book as a main textbook in graduate data science in education courses. We also wrote it as a practical reference for data scientists working with education data.
```

Comment:
Did we write it as a textbook?

Reply:
One of the use cases we imagined was professors using the book in a graduate school class. Leaving it as is, but no strong feelings about that. Happy to change it if others feel differently. + @joshuarosenberg and @efreer20